packages/cms/pages/settings/roles-and-permissions/roles-and-permissions.tsx

```diff
@@ -286,6 +310,7 @@ const RolesFlyout: React.FC<{
                 })
               }}
               name="name"
+              defaultValue={isEditing ? createRoleForm.name : ''}
```

Comment:
A better approach may be to have `defaultValue={createRoleForm.name}` here instead. Then, when initialising state, just make sure createRoleForm.name is `''` if creating and the name if editing.

Reply:
okay